presto-hive/src/main/java/io/prestosql/plugin/hive/HiveSessionProperties.java

```diff
@@ -467,6 +467,11 @@ public static boolean isUseParquetColumnNames(ConnectorSession session)
         return session.getProperty(PARQUET_USE_COLUMN_NAME, Boolean.class);
     }
 
+    /**
+     * @deprecated this can mask correctness issues
+     */
+    // TODO remove
+    @Deprecated
     public static boolean isFailOnCorruptedParquetStatistics(ConnectorSession session)
```

Comment:
Hi, @findepi, will turn this flag off cause correctness issue? We are deciding whether we should turn it on also but it seems like we will return `Domain.all()` if min>max but for hive connector we never trust pushed down predicate completely. The filter node on top of table scan node seems will filter those rows out (but there will be a performance hit)?